diff --git a/src/Cedar/Interop_OpenVPN.h b/src/Cedar/Interop_OpenVPN.h
index 20e53bb..2f7b5a8 100644
--- a/src/Cedar/Interop_OpenVPN.h
+++ b/src/Cedar/Interop_OpenVPN.h
@@ -139,7 +139,7 @@
 
 
 // The default algorithm
-#define	OPENVPN_DEFAULT_CIPHER					"AES-128-CBC"
+#define	OPENVPN_DEFAULT_CIPHER					"AES-256-CBC"
 #define	OPENVPN_DEFAULT_MD						"SHA1"
 
 // Encryption related
@@ -312,7 +312,7 @@ struct OPENVPN_SERVER_UDP
 };
 
 // OpenVPN Default Client Option String
-#define	OVPN_DEF_CLIENT_OPTION_STRING	"dev-type tun,link-mtu 1500,tun-mtu 1500,cipher AES-128-CBC,auth SHA1,keysize 128,key-method 2,tls-client"
+#define	OVPN_DEF_CLIENT_OPTION_STRING	"dev-type tun,link-mtu 1500,tun-mtu 1500,cipher AES-256-CBC,auth SHA1,keysize 128,key-method 2,tls-client"
 
 
 //// Function prototype
diff --git a/src/bin/hamcore/openvpn_sample.ovpn b/src/bin/hamcore/openvpn_sample.ovpn
index 63cc5b8..6b0aeff 100644
--- a/src/bin/hamcore/openvpn_sample.ovpn
+++ b/src/bin/hamcore/openvpn_sample.ovpn
@@ -85,7 +85,7 @@ $TAG_BEFORE_REMOTE$remote $TAG_HOSTNAME$ $TAG_PORT$
 #          RC2-40-CBC RC2-64-CBC RC2-CBC
 #  auth:   SHA SHA1 MD5 MD4 RMD160
 
-cipher AES-128-CBC
+cipher AES-256-CBC
 auth SHA1
 
 
